python/ray/serve/schema.py

```diff
@@ -311,91 +310,3 @@ class DeploymentStatusSchema(BaseModel, extra=Extra.forbid):
 
 class ServeApplicationStatusSchema(BaseModel, extra=Extra.forbid):
     statuses: List[DeploymentStatusSchema] = Field(...)
-
-
-def deployment_to_schema(d: Deployment) -> DeploymentSchema:
-    if d.ray_actor_options is not None:
-        ray_actor_options_schema = RayActorOptionsSchema.parse_obj(d.ray_actor_options)
-    else:
-        ray_actor_options_schema = None
-
-    return DeploymentSchema(
-        name=d.name,
-        import_path=get_deployment_import_path(d),
-        init_args=d.init_args,
-        init_kwargs=d.init_kwargs,
-        num_replicas=d.num_replicas,
-        route_prefix=d.route_prefix,
-        max_concurrent_queries=d.max_concurrent_queries,
-        user_config=d.user_config,
-        autoscaling_config=d._config.autoscaling_config,
-        graceful_shutdown_wait_loop_s=d._config.graceful_shutdown_wait_loop_s,
-        graceful_shutdown_timeout_s=d._config.graceful_shutdown_timeout_s,
-        health_check_period_s=d._config.health_check_period_s,
-        health_check_timeout_s=d._config.health_check_timeout_s,
-        ray_actor_options=ray_actor_options_schema,
-    )
-
-
-def schema_to_deployment(s: DeploymentSchema) -> Deployment:
-    if s.ray_actor_options is None:
-        ray_actor_options = None
-    else:
-        ray_actor_options = s.ray_actor_options.dict(exclude_unset=True)
-
-    return deployment(
-        name=s.name,
-        num_replicas=s.num_replicas,
-        init_args=s.init_args,
-        init_kwargs=s.init_kwargs,
-        route_prefix=s.route_prefix,
-        ray_actor_options=ray_actor_options,
-        max_concurrent_queries=s.max_concurrent_queries,
-        _autoscaling_config=s.autoscaling_config,
-        _graceful_shutdown_wait_loop_s=s.graceful_shutdown_wait_loop_s,
-        _graceful_shutdown_timeout_s=s.graceful_shutdown_timeout_s,
-        _health_check_period_s=s.health_check_period_s,
-        _health_check_timeout_s=s.health_check_timeout_s,
-    )(s.import_path)
-
-
-def serve_application_to_schema(
```

Comment:
Make sure to add `serve_application_to_schema` and `schema_to_serve_application` back to `api.py`. Otherwise, I believe `test_schema` will fail.